internal/pgbackrest/options.go

```diff
@@ -0,0 +1,96 @@
+/*
+ Copyright 2021 Crunchy Data Solutions, Inc.
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+
+package pgbackrest
+
+import (
+	"fmt"
+	"sort"
+	"strings"
+)
+
+// iniMultiSet represents the key-value pairs in a pgBackRest config file section.
+type iniMultiSet map[string][]string
+
+func (ms iniMultiSet) String() string {
+	keys := make([]string, 0, len(ms))
+	for k := range ms {
+		keys = append(keys, k)
+	}
+
+	sort.Strings(keys)
+
+	var b strings.Builder
+	for _, k := range keys {
+		for _, v := range ms[k] {
+			if len(v) <= 0 {
+				_, _ = fmt.Fprintf(&b, "%s =\n", k)
+			} else {
+				_, _ = fmt.Fprintf(&b, "%s = %s\n", k, v)
+			}
+		}
+	}
+	return b.String()
+}
+
+// Add associates value with key, appending it to any values already associated
+// with key. The key is case-sensitive.
+func (ms iniMultiSet) Add(key, value string) {
+	ms[key] = append(ms[key], value)
```

Comment:
:thinking: Should we care about duplicates when adding?

Reply:
I just checked, and pgBackRest allows duplicate values in at least one case.

Reply:
Ah very interesting ... thanks for confirming.

Reply:
Added to the doc in [e5f9d2c](https://github.com/CrunchyData/postgres-operator/compare/912f6c652aa820b7dc85a9e2218353b8409ab817..e5f9d2c32a83c59c31ab391fae9e94ab35d952f7).